tests/Functional/DatadirTest.php

```diff
@@ -234,7 +233,7 @@ public function testUndefinedAction(): void
             $configuration,
             1,
             null,
-            'Action \'funkyAction\' does not exist.' . PHP_EOL
```

Comment:
Jaktože to zmizelo?

Reply:
Prave kvuli NullHandleru pro sync akce.

Reply:
Ale nemělo by to vrátit JSON ve kterém je ta chyba nebo tak něco?

Reply:
Ne, stejne se nema jak zpracovat nebo kde zobrazit. Jen to vyhodi UserException.